text/testing/mock.md

```diff
@@ -1,21 +1,66 @@
 # Mock Runtime for Unit Testing
+
 *see [root](./index.md) for list of kitchen pallets with unit test coverage*
```

Comment:
I know this line existed prior to the PR, but is it still relevant? If not can you please remove it?

Reply:
I see not all pallets have unit test yet. So I think it is good to list out pallets with test, and readers could quickly refer back to these tests.